core/src/latest_unprocessed_votes.rs

```diff
@@ -361,6 +350,16 @@ mod tests {
         LatestValidatorVotePacket::new(packet, vote_source).unwrap()
     }
 
+    fn deserialize_packets<'a>(
+        packet_batch: &'a PacketBatch,
+        packet_indexes: &'a [usize],
+        vote_source: VoteSource,
+    ) -> impl Iterator<Item = LatestValidatorVotePacket> + 'a {
+        packet_indexes.iter().filter_map(move |packet_index| {
+            LatestValidatorVotePacket::new(packet_batch[*packet_index].clone(), vote_source).ok()
+        })
+    }
```

Comment:
This function seems unchanged, can we move this back where it was before to make the diff smaller